src/Target.h

```diff
@@ -258,6 +258,16 @@ struct Target {
     /** Was libHalide compiled with support for this target? */
     EXPORT bool supported() const;
 
+    /** Given a size (in bytes), return True if the allocation size can fit
```

Comment:
Target is a a way for the user to specify the target, not a database of everything codegen might want to know. Beyond this though, the codegen will likely want to tune the stack allocation decision based on other factors. E.g. if there are a ton of simultaneous allocations, one wants to make it smaller right?